### DISCUSSION TITLE
Notification system

### DISCUSSION THREAD

**Discussion by ociegonzalez** [dailydotdev/daily#1367] [Feature Requests]
Its possible to upgrade the notification systems of the app.

I find it kinda hard to see different types of notifications.

For example I post a lot of articles and I'll like to have a special space for notifications for my post only or comments I made in some post pls

**Comment by idoshamun** [comment]
Hi @soulis98,

Thanks for the suggestion, we will consider it

**Reply by ociegonzalez** [reply]
Ty!
